examples/lightgbm/train.py

```diff
@@ -29,31 +29,32 @@ def main():
     X = iris.data
     y = iris.target
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
-    dtrain = lgb.Dataset(X_train, label=y_train)
-
-    # train model
-    params = {
-        'objective': 'multiclass',
-        'num_class': 3,
-        'metric': 'multi_logloss',
-        'colsample_bytree': args.colsample_bytree,
-        'subsample': args.subsample,
-        'seed': 42,
-    }
-    model = lgb.train(params, dtrain)
-
-    # evaluate model
-    y_proba = model.predict(X_test)
-    y_pred = y_proba.argmax(axis=1)
-    loss = log_loss(y_test, y_proba)
-    acc = accuracy_score(y_test, y_pred)
-
-    # log parameters, metrics, and model
-    with mlflow.start_run() as run:
-        mlflow.log_params(params)
+    train_set = lgb.Dataset(X_train, label=y_train)
+
+    # enable auto logging
+    mlflow.lightgbm.autolog()
+
+    with mlflow.start_run():
```

Comment:
If we use autologging, we shouldn't need a `mlflow.start_run` anymore, right?

Reply:
If we didn't use `mlflow.start_run` here, `log_metrics` after `lgb.train` would create a new run and record metrics in it.